sdk/maps/azure-maps-service/samples/data.py

```diff
@@ -0,0 +1,107 @@
+# These examples are ingested by the documentation system, and are
+# displayed in the SDK reference documentation. When editing these
+# example snippets, take into consideration how this might affect
+# the readability and usability of the reference documentation.
+import argparse
+from typing import List
+import sys
+import json
+import zipfile
+
+from azure.maps.service.operations import DataOperations
+from azure.maps.service.models import UploadDataFormat, MapDataListResponse, MapDataDetailInfo
+from azure.core.exceptions import HttpResponseError
+from common.common import create_maps_client, get_operation_location_id, wait_for_status_complete
+
+import gzip
+from io import StringIO, BytesIO
+
+
+def upload(data: DataOperations):
+    with open("resources/data_sample_upload.json", "r") as file:
+        poller = data.begin_upload_preview(UploadDataFormat.GEOJSON,
+                                           json.loads(file.read()), "Upload Description", polling=True)
```

Comment:
Why not just call `json.load`?

Reply:
...and we should not pass in the polling parameter. It is intended to override the default behavior, which we are not doing here...